oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -117,13 +126,27 @@ private ComputeEngineCredentials(HttpTransportFactory transportFactory) {
    * @return new ComputeEngineCredentials
    */
   public static ComputeEngineCredentials create() {
-    return new ComputeEngineCredentials(null);
+    return new ComputeEngineCredentials(ImmutableSet.<String>of(), null);
```

Comment:
Should this default to null instead?

An empty collection of scopes might make the url something like `https://some-url/?scopes=` which may be correct if you ask for no scopes, but if you pass null, it should exclude the scopes param.

Reply:
Yes, you're right.  I originally wrote this so that an empty collection would result in not sending the `?scopes` param, but then changed it to null.

Reply:
Done